security-analysis/security-analysis-api/src/main/java/com/powsybl/security/scenario/ScenarioResult.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.scenario;
+
+import com.google.common.collect.ImmutableMap;
+import com.powsybl.security.LimitViolationsResult;
+import com.powsybl.security.results.BranchResult;
+import com.powsybl.security.results.BusResults;
+import com.powsybl.security.results.ThreeWindingsTransformerResult;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot <etienne.lesot@rte-france.com>
+ */
+public class ScenarioResult {
+
+    private final Scenario scenario;
+
+    private final LimitViolationsResult limitViolationsResult;
+
+    private final Map<String, BranchResult> branchResults;
+
+    private final Map<String, BusResults> busResults;
+
+    private final Map<String, ThreeWindingsTransformerResult> threeWindingsTransformerResults;
+
+    public ScenarioResult(Scenario scenario, LimitViolationsResult limitViolationsResult, Map<String, BranchResult> branchResults, Map<String, BusResults> busResults, Map<String, ThreeWindingsTransformerResult> threeWindingsTransformerResults) {
```

Comment:
Would probably be better to accept collections as input, instead of maps.
The mapping will be done by the constructor.